### PR TITLE
DM-20566: Replace afwGeom with geom

### PR DESCRIPTION
**Pull request by timj**
There are some flake8 fixes as well but the package has some fundamental problems with variables being used that are not defined.

### CODE REVIEW

File: tests/test_PsfexPsf.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 import lsst.afw.image as afwImage
 import lsst.afw.detection as afwDetection
 import lsst.afw.geom as afwGeom
+import lsst.geom as geom
```
Comment (parejkoj) [inline tests/test_PsfexPsf.py]: You couldn't also remove the `afwGeom` above?

Comment (timj) [reply tests/test_PsfexPsf.py]: No, because `makeSkyWcs` is used here.

**Comment by timj** [comment]
I've had a look at the 4 problematic areas. Fixes are not obvious so I've put in exceptions (they would throw any how) to be a bit more explicit about it.

**Comment by timj** [comment]
I'm not sure of anything, but since they are branches that people *might* encounter even if we removed the code in the branch we should still retain the exception raising rather than ignoring the configuration option. I'm therefore inclined to leave it for now.

**Pull request merged**